source/content/disaster-recovery.md

```diff
@@ -44,8 +44,27 @@ You can also [connect to your Redis instance](/redis/#use-the-redis-command-line
 
 If you rely on the Redis cache for locks (mutexes) or storing other long-term data, you must move them out of Redis and into the database to avoid any issues when the Redis cache is dropped during failover.
 
-## Limitations
-Solr is not currently supported on DR-enabled sites. See our documentation on how to safely remove Solr for [Drupal 8](/solr-drupal-8/#safely-remove-solr), [Drupal 7](/solr-drupal-7/#safely-remove-solr), and [WordPress](/wordpress-solr/#safely-remove-solr).
+## Solr and Disaster Recovery
+
+### Considerations for Sites Requiring Highly Available Solr Service
+
+While Solr can be enabled on a site with Disaster Recovery, Pantheon's Solr service is not highly available, nor does it include failover for Solr. The RTO and RPO do not apply to Pantheon's Solr Service.
+
+If your site requires Solr for basic functionality, do not use Pantheon's Solr service. If you require a highly available Solr service, please consider an [alternative Solr service](/solr#alternatives-to-pantheons-solr-service).
```

Comment:
basic functionality --> . critical site functionality  ?

Reply:
IMHO I'd just mare it "requires Solr".

Reply:
Resolved by https://github.com/pantheon-systems/documentation/pull/5251/commits/5df8c99e6fcc1ecd40dbf192f01efba2e4a930a8